src/dataview64.js

```diff
@@ -54,6 +64,18 @@ export default class DataView64 {
     return this._dataView.getInt32(offset, littleEndian);
   }
 
+  getUint40(offset, littleEndian) {
```

Comment:
I'm not sure if we actually need this function. Also, the big endian case has to be handled as-well.